.allowed-scripts.mjs

```diff
@@ -0,0 +1,18 @@
+import { configureAllowedScripts } from '@ministryofjustice/hmpps-npm-script-allowlist'
+
+export default configureAllowedScripts({
+  allowlist: {
```

Comment:
These allowed dependencies were taken from:

https://github.com/ministryofjustice/hmpps-template-typescript/commit/98735f32c3ed9f57b16d14398810d89b954c208b#diff-be0b106dba7c5a3635614a4e23e4e29c9da4462922769c8f99b60e31d288c7b1

I cannot take responsibility for whether they are correct or not.